notes/returned-mail.vbs

```diff
@@ -268,44 +290,45 @@ IF ADDR_actions <> "no response received" THEN
 		If DateDiff("d", begining_of_footer_month, addr_eff_date) > 0 Then begining_of_footer_month = addr_eff_date
 		Call access_ADDR_panel("WRITE", notes_on_address, resi_addr_line_one, resi_addr_line_two, resi_street_full, resi_addr_city, resi_addr_state, resi_addr_zip, county_code, addr_verif, homeless_addr, reservation_addr, living_situation, reservation_name, new_addr_line_one, new_addr_line_two, new_addr_street_full, new_addr_city, new_addr_state, new_addr_zip, begining_of_footer_month, addr_future_date, phone_one, phone_two, phone_three, type_one, type_two, type_three, text_yn_one, text_yn_two, text_yn_three, addr_email, verif_received, original_information, update_attempted)
 	END IF
-ELSE
-    Dialog1 = "" 'Blanking out previous dialog detail
-    BeginDialog Dialog1, 0, 0, 351, 160, "Request for Contact to the client with no response-Refused/Failed"
-     EditBox 105, 5, 50, 15, date_verifications_requested
-     ButtonGroup ButtonPressed
-       PushButton 5, 140, 65, 15, "PACT TE02.13.10", POLI_TEMP_PACT_button
-       OkButton 240, 140, 50, 15
-       CancelButton 295, 140, 50, 15
-     Text 5, 10, 100, 10, "Date Verifications Requested:"
-     Text 5, 25, 305, 10, "Allow 10 days for the client to respond to the Verification Request before terminating benefits."
-     Text 5, 40, 330, 10, "Approve ineligible results in ELIG. Send a closing notice 10 days before the effective date of closing."
-     Text 5, 55, 340, 35, "Remember to enter a worker comment in SPEC/WCOM to add a detailed explanation of closure to the notice.  DHS suggested text for the SPEC/WCOM: Your mail has been returned to our agency.  On (insert date) you were sent a request to contact this agency because of this returned mail.  You can avoid having your case closed if you contact this agency by (insert the closing date deadline)."
-     Text 20, 95, 295, 10, "SNAP the script will enter code 4 when mail to the client has been returned to the agency."
-     Text 20, 110, 265, 10, "CASH the script will enter code 3 in the Close/Deny field on the STAT/PACT Panel."
-     Text 20, 125, 225, 10, "Health Care cannot be denied at this time for whereabouts unknown."
-    EndDialog
+END IF 'forwarding address provided
+IF ADDR_actions = "no response received" THEN
```

Comment:
TIKL is only being set for 'ADDR_actions = "no response received"' because it's in the If...statement.
TIKL doesn't get set if date field is filled out in main dialog (TIKL code on Line 370).

Reply:
Default Closing message on line 421 says TIKL has been sent for ADDR verification. This is not the case in all circumstances - mail found in error as example.

Reply:
Created IF statements for each addr_action option